api/python/cellxgene_census/tests/experimental/diffexp/memento/test_diff_expr.py

```diff
@@ -1,156 +1,97 @@
-import itertools
-from typing import Dict
+from os import path
+from typing import Any, Dict
 
+import numpy as np
 import pandas as pd
 import pytest
 
 from cellxgene_census.experimental.diffexp.memento import diff_expr
-from cellxgene_census.experimental.diffexp.memento.diff_expr import CUBE_LOGICAL_DIMS_OBS
 
 
-@pytest.fixture(scope="function")
-def estimators_df(dim_counts: Dict[str, int]) -> pd.DataFrame:
-    """Create a dummy estimators DataFrame, with all combinations of variables."""
-    columns = CUBE_LOGICAL_DIMS_OBS + ["feature_id"]
-    dim_counts_full = {var: 1 for var in columns}
-    dim_counts_full.update(dim_counts or {})
-    dim_values = {var: [f"{var}_{chr(ord('a') + i)}" for i in range(n)] for var, n in dim_counts_full.items()}
+class TestDiffExprRealDataset:
+    """
+    This class contains regression tests that run on realistic datasets.
 
-    rows = list(itertools.product(*dim_values.values()))
-    estimators = pd.DataFrame(data=rows, columns=columns).astype("str")
+    Since this class is intended to be serve as a regression test suite
+    using real data, it is strongly recommended that the functions under
+    test are all PUBLIC functions. Private functions are best encapsulated
+    in separate class.
+    """
 
-    for c, estimator in enumerate(["n_obs", "mean", "sem", "var", "selv"], start=1):
-        estimators[estimator] = range(1, len(estimators) + 1)
-        estimators[estimator] *= c
+    @pytest.fixture(scope="class", params=["test_case_1", "test_case_2"])
+    def test_cases_for_compute_all_fn(self, request: Any) -> Dict[str, Any]:
```

Comment:
decorate all test functions with `@pytest.mark.experimental`, since these are all for experimental code (we probably should rethink this, since all tests are under /experimental dir, but it's the repo's convention for now).